File: edX/Harvard CS50/pset6/mario/more/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

counter = 0

for i in range(h):
    counter += 1
    for j in range(h - 1):
        print(" ", end="")
    for k in range(counter):
        print("#", end="")

    print("  ", end="")

    for l in range(counter):
        print("#", end="")

    print()
    h -= 1

File: edX/Harvard CS50/pset1/mario2/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Declare integers
    int h, r, c, b;

    //The loop that forces the user to give a number between 1 and 8
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    //Row loop, outer most
    for (r = 0; r < h; r++)
    {

        //Blank loop
        for (b = 0; b < h - r - 1; b++)
        {
            printf(" ");
        }

        //Hash loops
        for (c = 0; c <= r; c++)
        {
            printf("#");
        }

        printf("\n");

    }

}